src/NuGetForUnity/Editor/NugetPackageTextureHelper.cs

```diff
@@ -117,9 +117,9 @@ private static string GetHash(string s)
             var md5 = new MD5CryptoServiceProvider();
             var data = md5.ComputeHash(Encoding.Default.GetBytes(s));
             var sBuilder = new StringBuilder();
-            for (var i = 0; i < data.Length; i++)
+            foreach (var dataByte in data)
             {
-                sBuilder.Append(data[i].ToString("x2"));
+                sBuilder.Append(dataByte.ToString("x2"));
```

Comment:
Is `Convert.ToHexString` not available in Unity 2018?

Reply:
Unfortunately not:
![image](https://github.com/GlitchEnzo/NuGetForUnity/assets/53140583/2cd16d27-9025-4cda-8d2c-025f57d90cef)
I changed it to `Convert.ToBase64String` witch even produces shorter strings. It invalidates the current hashes but I see no problem when it changes, it are only images that are stored inside `Application.temporaryCachePath`.